tools/build_third_party_libs.py

```diff
@@ -27,7 +27,7 @@
 
     if args.c_compiler is None:
         if sys.platform == "win32":
-            c_compiler = "cl"
+            c_compiler = '"cl -std:c++20"'
```

Comment:
Is this the correct flag for Visual Studio? I always need to use different flags on Compiler Explorer whenever I'm testing MSVC.

Reply:
Aah, I'm not sure how that compiled! It should be `/std:c++20` and not `-std:c++20`. I updated code to add another variable called `cxx_flags`.